assets/scripts/utils/is.js

```diff
@@ -1,37 +1,67 @@
 const toString = Object.prototype.toString;
 const arrayLikePattern = /^\[object (?:Array|FileList)\]$/;
 
-// thanks, http://perfectionkills.com/instanceof-considered-harmful-or-how-to-write-a-robust-isarray/
-export function isArray ( thing ) {
-    return toString.call( thing ) === '[object Array]';
-}
+/**
+ * Check if argument is an array
+ * @param {?array}      x   - element to check
+ * @return {boolean}    true if element is an array
+ */
 
-export function isArrayLike ( obj ) {
-    return arrayLikePattern.test( toString.call( obj ) );
-}
+const isArray = x => toString.call(x) === '[object Array]'
 
-export function isEqual ( a, b ) {
-    if ( a === null && b === null ) {
-        return true;
-    }
 
-    if ( typeof a === 'object' || typeof b === 'object' ) {
-        return false;
-    }
+/**
+ * Check if argument is an array like
+ * @param {array|object}    x   - element to check
+ * @return {boolean}        true if element is an array like
+ */
 
-    return a === b;
-}
+const isArrayLike = x => arrayLikePattern.test(toString.call(x))
```

Comment:
This expands the scope of this function beyond `Array` and `FileList` by checking for `length` which is the primary feature of "array-like" values.

```suggestion
const isArrayLike = x => (x && typeof x !== 'function' && isArrayLikeLength(x.length))

/**
 * Checks if argument is a valid array-like length.
 *
 * @param   {*} value - The value to check.
 * @returns {boolean}
 */
const isArrayLikeLength = x => (typeof x === 'number' && x > -1 && x % 1 == 0)
```